ansible-ipi-install/inventory/jetski/hosts

```diff
@@ -109,20 +113,24 @@ cluster="test"
 # when redeploying an existing environment this can help avoid VRID conflicts. You can
 # set the cluster_random boolean below to true to append a random number to you cluster name.
 cluster_random=true
-# The public CIDR address, i.e. 10.1.1.0/21
-extcidrnet="192.168.216.0/21"
+# The public CIDR address, i.e. 10.1.1.0/21 in list format, containing only one entry for
+# IPv6 only or IPv4 only baremetal network. List contains two elements, with the first entry
+# being the IPv4 range in the case of dual-stack. Example for dual-stack is
+# extcidrnet=["192.168.216.0/21", "fd01:1101::/64"]. extcidr is a lit with atleast one element
+# and not more than two elements.
+extcidrnet=["192.168.216.0/21", "fd01:1101::/64"]
```

Comment:
Just a thought, in case if you wanna ignore this change, let IPv6 be hard coded in script as its not used in many places. Only at bridge creation and dnsmasq task right.